learn/guides/ballerina-openapi-support.md

```diff
@@ -24,28 +24,30 @@ redirect_from:
   - /learn/cli-documentation/openapi
 ---
 
-## Use the capabilities of the OpenAPI tools
-
 The Ballerina OpenAPI tooling support provides the following capabilities.
+
+> **Prerequisite**: Download and install the Ballerina Swan Lake from [downloads](https://ballerina.io/downloads/).
+
  1. Generating Ballerina service/client stubs from a given OpenAPI contract file using the CLI tool.
  2. Exporting the OpenAPI definition from a given Ballerina service implementation using the CLI tool.
  3. Validating the service implementation compliance with a provided OpenAPI contract using the OpenAPI annotation.
     - The OpenAPI compiler plugin will allow you to validate a service implementation against an OpenAPI contract during
   the compile time. This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract. 
-    
- > **Prerequisite**: Download and install the Ballerina Swan Lake from [downloads](https://ballerina.io/downloads/).
 
- > **Note**: Ballerina SwanLake supports OpenAPI Specification version 3.0.0 onwards.
+ > **Note**: Ballerina Swan Lake supports the OpenAPI Specification version 3.0.0 onwards.
 
-## Generate a Ballerina service from an OpenAPI definition (design-first approach)
+## Generate a Ballerina service 
+
+Generating a Ballerina service from an OpenAPI definition is named as the `design-first approach`. To generate only the Ballerina service, you can use the following CLI command of the OpenAPI tool.
```

Comment:
Following the  `API design-first approach`,  you can generate only the Ballerina service from an OpenAPI definition, using the below CLI command of the OpenAPI tool.

Reply:
```suggestion
Following the  `API design-first approach`,  you can generate only the Ballerina service from an OpenAPI definition, using the below CLI command of the OpenAPI tool.
```